src/client/sky.cpp

```diff
@@ -334,25 +339,28 @@ void Sky::render()
 				vertices[i * 4 + 3].Color = starcolor;
 			}
 			driver->drawVertexPrimitiveList(vertices, SKY_STAR_COUNT * 4,
-				indices, SKY_STAR_COUNT, video::EVT_STANDARD,
-				scene::EPT_QUADS, video::EIT_16BIT);
+					indices, SKY_STAR_COUNT, video::EVT_STANDARD,
+					scene::EPT_QUADS, video::EIT_16BIT);
 #endif
-		} while(false);
+		} while (false);
 
-		// Draw sunrise/sunset horizon glow texture (textures/base/pack/sunrisebg.png)
+		// Draw sunrise/sunset horizon glow texture
+		// (textures/base/pack/sunrisebg.png)
```

Comment:
Please only wrap lines at 90 columns (tab size 4). This may be a bad clang format suggestion.